collectors/apps.plugin/apps_plugin.c

```diff
@@ -107,6 +107,26 @@ static int
 static char *user_config_dir = CONFIG_DIR;
 static char *stock_config_dir = LIBCONFIG_DIR;
 
+// some variables for keeping track of processes count by states
+#define NETDATA_CHART_PRIO_SYSTEM_PROCESS_STATES 601
```

Comment:
Please, instead to add this definition here, move it after [this line](https://github.com/netdata/netdata/blob/e3a1087b9f5200e670a22bb7af54573d40167416/collectors/all.h#L32), to avoid conflict in he future.

Reply:
agree